app/components/Views/SmartTransactionsOptInModal/SmartTransactionsOptInModal.test.tsx

```diff
@@ -96,7 +96,7 @@ describe('SmartTransactionsOptInModal', () => {
       Engine.context.PreferencesController.setSmartTransactionsOptInStatus,
     ).toHaveBeenCalledWith(true);
   });
-  it('should opt user out when secondary button is pressed and navigate to Advanced Settings', () => {
+  it('should opt user out when secondary button is pressed and navigate to Advanced Settings', async () => {
```

Comment:
Please check contributor guidelines for test naming:
```suggestion
  it('opts user out when secondary button is pressed and navigate to Advanced Settings', async () => {
```